db/migrate/20160319110950_update_user_reputation_from_existing_votes.rb

```diff
@@ -0,0 +1,31 @@
+class UpdateUserReputationFromExistingVotes < ActiveRecord::Migration
+  def up
+	  User.find_each{|user| user.increment!(:points, 10)}
+
+	  Vote.all.each do |vote|
+      subject = vote.voteable_type.constantize
+      value = vote.value
+      reward = vote.evaluate_reward(true, value, subject)
+      subject_id = vote.voteable_id
+      user = subject.find_by(id: subject_id).user
+      user.update_user_reputation(reward) if reward
+    end
+
+    Answer.all.select(&:accepted).map(&:user).each{|u| u.update_user_reputation(20)}
```

Comment:
This is an expensive query. You are fetching all answers before selecting the accepted ones. Then you went ahead to select the user for each accepted answer thereby introducing `N + 1` query. Queries in Migration needs to be fast. I might even go as far as using raw sql.
```ruby
execute <<-SQL
  UPDATE users u
  SET u.points = u.points + 20, 
  FROM 
    (SELECT mu.id 
     FROM user mu 
     JOIN answers a
     ON a.user_id = mu.id
     WHERE a.accepted = true) o
WHERE 
    o.id = u.id
SQL
```
Try it out and let me know if it works. Well if it doesn't, you can make it work. :smile: